lib/ansible/executor/task_queue_manager.py

```diff
@@ -64,6 +66,21 @@ def __init__(self, *args, **kwargs):
         self.kwargs = kwargs
 
 
+@dataclass
+class PromptSend:
+    worker_id: int
+    prompt: str
+    private: bool = True
+    seconds: int = None
+    interrupt_input: t.Iterable[bytes] = None
+    complete_input: t.Iterable[bytes] = None
+
+    kwargs: t.Mapping = field(init=False, default_factory=dict)
```

Comment:
Yeah, thinking about it a little more, this is just one of those reflexive design red flags that actually doesn't hurt in this specific case (it's a really brittle way to call a quasi-related method in the general case). OTOH, since there's only one caller, it's probably safer and simpler to just use the public attrs directly at the call-site- the end result is net less code anyway and one less thing you have to worry about keeping in sync.

Reply:
I updated it to use the public attrs at the call site.